Aardvark/ARKExceptionLogging.h

```diff
@@ -0,0 +1,28 @@
+//
+//  ARKExceptionLogging.h
+//  Aardvark
```

Comment:
Should this be a part of `CoreAardvark`? This seems like core functionality, but I'm not sure if the APIs we're using here are extension safe.

Reply:
I think it should be fine in `CoreAardvark`.  I'll do some testing, but don't see anything that says it isn't extension safe.